dds/DdsDcpsCore.idl

```diff
@@ -272,22 +272,27 @@ module DDS {
       DISALLOW_TYPE_COERCION,
       ALLOW_TYPE_COERCION
     };
-
+    */
+    //constants for TypeConsistencyEnforcementQosPolicy::kind field
+    const short DISALLOW_TYPE_COERCION = 1;
+    const short ALLOW_TYPE_COERCION = 2;
```

Comment:
Use a typedef for `short` (both here and in the struct) to make it clear what this is.